examples/multiplatform-example/android/src/test/java/com/karumi/hagu/AndroidSharedTest.kt

```diff
@@ -4,7 +4,7 @@ import com.karumi.hagu.generated.HaguConfig
 
 import org.junit.Test
 import junit.framework.TestCase.assertEquals
-
+//- ./gradlew clean test --tests "*CIProfileTest" -Pprofile=ci --info
```

Comment:
I guess you want to remove this comment. Am I wrong?

Reply:
oh god -.-, that is because I have three different projects opened